package.json

```diff
@@ -35,14 +35,14 @@
   ],
   "dependencies": {
     "@babel/runtime": "^7.3.1",
+    "@types/seamless-immutable": "6.1.2",
```

Comment:
think the types should match the dependency version. a bit of a pain to make sure we're using the same version.

Reply:
Thats a good point.